cmd/ops_agent_uap_plugin/service_windows.go

```diff
@@ -29,14 +36,7 @@ func (ps *OpsAgentPluginServer) Apply(ctx context.Context, msg *pb.ApplyRequest)
 // Until plugin receives Start request plugin is expected to be not functioning
 // and just listening on the address handed off waiting for the request.
 func (ps *OpsAgentPluginServer) Start(ctx context.Context, msg *pb.StartRequest) (*pb.StartResponse, error) {
-	if ps.cancel != nil {
-		log.Printf("The Ops Agent plugin is started already, skipping the current request")
-		return &pb.StartResponse{}, nil
-	}
-	log.Printf("Received a Start request: %s. Starting the Ops Agent", msg)
-
-	_, cancel := context.WithCancel(context.Background())
-	ps.cancel = cancel
+	// Unimplemented
```

Comment:
Should we panic("Unimplemented functionality") here instead of returning something invalid?

Reply:
Done